examples/nidmm_measurement/teststand_fixture.py

```diff
@@ -55,22 +56,13 @@ def create_nidmm_sessions(sequence_context: Any) -> None:
             timeout=0,
         ) as reservation:
             for session_info in reservation.session_info:
-                options: Dict[str, Any] = {}
-                if USE_SIMULATION:
-                    options["simulate"] = True
-                    options["driver_setup"] = {"Model": "4081"}
-
-                grpc_options = nidmm.GrpcSessionOptions(
-                    grpc_channel_pool.get_grpc_device_channel(nidmm.GRPC_SERVICE_INTERFACE_NAME),
-                    session_name=session_info.session_name,
-                    initialization_behavior=nidmm.SessionInitializationBehavior.INITIALIZE_SERVER_SESSION,
+                grpc_device_channel = grpc_channel_pool.get_grpc_device_channel(
+                    nidmm.GRPC_SERVICE_INTERFACE_NAME
                 )
-
-                # Leave session open
-                nidmm.Session(
-                    resource_name=session_info.resource_name,
-                    options=options,
-                    grpc_options=grpc_options,
+                create_session(
+                    session_info,
+                    grpc_device_channel,
```

Comment:
Overlooked in NI-DCPower PR review: where did the comment go? Also, a discard would make it more explicit that we are ignoring the return value.

```
# Leave session open
_ = create_session(
```